modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -0,0 +1,77 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
```

Comment:
This is generally the full path to the module. We don't seem to have this file? I think you meant:

```
* installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
```